### PR TITLE
Users can create/update organizations

### PR DESCRIPTION
**Pull request by mauricioabreu**

### CODE REVIEW

**Comment by mauricioabreu** [comment]
It fixes #130 and #286

**Comment by mauricioabreu** [comment]
After having the `list` we can include some links, etc

**Comment by mauricioabreu** [comment]
OMFG why is it still being tested on TravisCI lol? @luanfonceca  :((( save us

File: organization/tests/test_functional.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+        data = {'name': 'Speakerfight Corp', 'about': 'Cool company'}
+        url = reverse('create_organization')
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(
+            response,
+            reverse('update_organization', kwargs={'slug': 'speakerfight-corp'})
+        )
+        self.assertEqual(Organization.objects.count(), 1)
```
Comment (luanfonceca) [inline organization/tests/test_functional.py]: Can you please assert the organization name and description?

Comment (mauricioabreu) [reply organization/tests/test_functional.py]: For sure \o

**Comment by mauricioabreu** [comment]
I missed the translation files 👍

**Comment by mauricioabreu** [comment]
I am having a bad time installing gettext to compile translation messages. :(

File: organization/tests/test_functional.py (hunk 1 above)
Comment (mauricioabreu) [reply organization/tests/test_functional.py]: Done

File: organization/views.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from django.core.urlresolvers import reverse
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
+from django.views.generic.edit import CreateView, UpdateView
+
+from . models import Organization
+from core.mixins import LoginRequiredMixin, FormValidRedirectMixing
+
+
+class CreateOrganization(LoginRequiredMixin,
```
[the remark this answers is not in the document]
Comment (mauricioabreu) [reply organization/views.py]: About the name ok.
About the `BaseOrganizationView` I find it a bad practice :( But ok, I can change that

File: organization/views.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    def get_success_url(self):
+        return reverse('update_organization', kwargs={'slug': self.object.slug})
+
+
+class UpdateOrganization(LoginRequiredMixin,
+                         UpdateView,
+                         FormValidRedirectMixing):
+    model = Organization
+    fields = ['name', 'about']
+    template_Name = 'organization/organization_form.html'
```
[the remark this answers is not in the document]
Comment (mauricioabreu) [reply organization/views.py]: Nope. Django tries to find if it has pk or slug I guess so.

File: organization/views.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    def form_valid(self, form):
+        self.object = form.save()
+        return self.success_redirect(_(u'Organization updated.'))
```
[the remark this answers is not in the document]
Comment (mauricioabreu) [reply organization/views.py]: I know but I am having problems with gettext :P

File: organization/models.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+from django_extensions.db.fields import AutoSlugField
+
+
+@python_2_unicode_compatible
+class Organization(models.Model):
+    name = models.CharField(_('Name'), max_length=100)
+    slug = AutoSlugField(populate_from='name', overwrite=True,
+                         max_length=200, unique=True, db_index=True)
+    about = models.TextField(
```
[the remark this answers is not in the document]
Comment (mauricioabreu) [reply organization/models.py]: Ugh, ok haha

File: organization/views.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mauricioabreu) [reply organization/views.py]: Where is the `Name` in organization_form.html? Can you point that in the code? I could not find it :(

**Pull request merged**